proposals/acp/wac-acp-diff-story.md

```diff
@@ -0,0 +1,127 @@
+# ACP Compared to Web Access Control
+
+Audience: those familiar with WAC or those who learn from comparisons
+
+Access control is fundamentally about stating `who` can `do what` `to what` resource.
+
+## ShEx for WAC:
+Web Access Control (WAC) uses a simple schema to make those assertions directly in RDF:
+
+``` shex
+PREFIX acl: <http://www.w3.org/ns/auth/acl#>
+PREFIX foaf: <http://xmlns.com/foaf/0.1/>
+
+<AclShape> {
+  # who
+  (  acl:agentClass @<GroupShape> OR [foaf:Agent] ;
+    | acl:agent IRI
+    | acl:origin IRI
+  )+ ;
+
+  # can do what
+  acl:mode [acl:Read acl:Write acl:Append acl:Control] + ;
+
+  # to what resource
+  (   acl:accessTo IRI
+    | acl:accessToClass [foaf:Document] OR @<AclRegexShape>
```

Comment:
I am not sure enough about this shape language. But it is saying that acl:accessToClass is either foaf:Document or an a RegexShape. I used regexes in rww-play, but I don't think the current WAC does this. (I may be mistaken). Still a regex can be used as a way among others of constraining a set resources. Other ways are by describing creation date, authorship, etc... So if you cut out the subclassing possibilities of RDF then you will come to conclusions about the limitations of WAC that are related to the constraint language used, not to RDF itself.